docker/nginx-cors.conf

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
_Same as before (and for all the `.conf` files)_ 😅

Do we actually need this?
As of 0.24.0 CORS are included in Epoch.